UndertaleModTool/Windows/SettingsWindow.xaml

```diff
@@ -60,41 +61,44 @@
             <CheckBox Grid.Row="5" Grid.Column="2" Margin="3" Content="" IsChecked="{Binding ShowDebuggerOption}"/>
             <TextBlock Grid.Row="5" Grid.Column="2" Margin="25 2 2 2" Text="Show &quot;Run game under GMS debugger&quot; file option" ToolTip="Whether to show the option in the &quot;File&quot; menu. Disabled by default."/>
 
-            <Separator Grid.Row="6" Grid.ColumnSpan="4" Margin="10"/>
+            <CheckBox Grid.Row="6" Grid.Column="0" Margin="3" Content="" IsChecked="{Binding HideChildCodeEntries}"/>
+            <TextBlock Grid.Row="6" Grid.Column="0" Margin="25 2 2 2" Text="Hide child code entries" ToolTip="Hides code entries corresponding to sub-functions of scripts in GMS2.3 games."/>
```

Comment:
Can you change it so that the text and/or tooltip mentions that this is for the visible tree-view thingy?

Reply:
I wasn't sure how to call it so users would understand so I called it "the asset list". In the tooltip for "Enable asset order swapping" it's called "the asset tabs" but that seems confusing given that there are no tabs in a tree view and now UTMT has tabs for content.